Connect/AzureStack.Connect.psm1

```diff
@@ -55,10 +55,14 @@ function Get-AzureStackAadTenant
     return Invoke-Command -ComputerName "$HostComputer" -Credential $credential -ScriptBlock `
         {            
             Write-Verbose "Retrieving Azure Stack configuration..." -Verbose
-            $config = Get-Content -Path "C:\CloudDeployment\Config.xml" -ErrorAction Stop
-            
-            ([xml]($config)).SelectSingleNode("//Role[@Id='AAD']").PublicInfo.AADTenant.Id
-
+            $configFile = Get-ChildItem -Path C:\EceStore -Recurse | ?{-not $_.PSIsContainer} | sort Length -Descending | select -First 1
```

Comment:
Reading from EceStore may not work in future,  Can you please add a comment saying this might change in future? Are we also assuming that this script is run from the Host/DVM?